docs/extending.md

```diff
@@ -85,7 +85,7 @@ CMS.registerEditorComponent({
   // Fields the user need to fill out when adding an instance of the component
   fields: [{name: 'id', label: 'Youtube Video ID', widget: 'string'}],
   // Pattern to identify a block as being an instance of this component
-  pattern: youtube (\S+)\s,
+  pattern: /^\s?youtube (\S+)\s?/,
```

Comment:
I think using the Hugo style delimiters (`{{<`/`>}}`) makes sense here for example purposes - any reason to exclude them?